docs/questions/start.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Questions overview
+---
+
+# Questions
+
+Questions are queries plus their visualization. You can ask questions using Metabase's graphical query builder, or create a native/SQL query.
+
+## Query builder
+
+### [Asking questions](./query-builder/introduction.md)
+
+An introduction to Metabase's graphical query builder.
+
+### [Custom expressions](./query-builder/expressions.md)
+
+Custom expressions, like spreadsheet formulas, are tools you can use in the graphical query builder to ask more complicated questions.
+
+### [List of expressions](./query-builder/expressions.md)
+
+A list of the aggregations and functions available in the graphical query builder.
+
+### [Joining data](./query-builder/join.md)
+
+You can join data to combine your current data with another table, or even with a saved question.
+
+## Native query editor
+
+Also known as the SQL editor (we say native because you can also query data sources that don't use SQL, like MongoDB and Google Analytics).
+
+### [Introduction to the native query editor](./native-editor/writing-sql.md)
+
+Write native code to query your data source.
+
+### [Viewing metadata](./native-editor/data-model-reference.md)
+
+Metabase's data model reference can help you understand your tables and fields.
+
+### [SQL templates](./native-editor/sql-parameters.md)
```

Comment:
Maybe a question for later, but... "templates", "variables", or "parameters"?

In theory, you can do stuff like, "pass the parameter to the variable in your template" (kinda textbook-y, but at least the flow of info is clear).

I feel like the "feature" should have a consistent name that refers to its behavior, and then the "parts" of that feature (parameter, variable, template) could get defined within it.

But yeah, for later (probably for product, too) 😁